bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClient.java

```diff
@@ -511,12 +538,14 @@ public void safeRun() {
         }
     }
 
-    public boolean isClosed() {
-        return closed;
+    public void monitorPendingOperations() {
+        for (PerChannelBookieClientPool clientPool : channels.values()) {
```

Comment:
channels.values().forEach ?

Reply:
I don't think it would add anything to the clarity, and we'd be allocating a lamba each time.